prefs.js

```diff
@@ -34,7 +34,7 @@ const pretty_names = {
 	'show-toggle-tiling': 'Display gTile',
 	'set-tiling'	    : 'Set tiling',
 	'cancel-tiling'     : 'Cancel tiling',
-	'change-grid-size'  : 'Change grid size',
+    'change-grid-size'  : 'Change grid size',
```

Comment:
Looks like the indentation got messed up here. This should be a tab so it conforms with the rest of the file. You might have an editor setting to fix that is changing indentation in some cases.

Reply:
Pushed commit with indentation fix, please look on PR.